tuneme/templates/core/tags/section_listing_homepage.html

```diff
@@ -26,7 +26,7 @@ <h2 class="list-header">{% trans "Latest" %}</h2>
           <p>{{page.subtitle}}</p>
           {% get_comment_count for page as comment_count %}
           {% if comment_count > 0 %}
-            <div class="comments-count"><a href="#">{{comment_count}} <span>{% trans "comment" %}{{comment_count|pluralize}}</span></a></div>
+            <div class="comments-count"><a href="{% pageurl page %}">{{comment_count}} <span>{% trans "comment" %}{{comment_count|pluralize}}</span></a></div>
```

Comment:
@TrevorOctober a suggestion here to link it directly to the comments section. It'll need 2 things
1. add an `id` to the article page where the [comments section starts](https://github.com/praekelt/molo-tuneme/blob/feature/issue-73-yep-76/tuneme/templates/core/article_page.html#L88) You can add the following div around all the comments `<div id="comments-list">`
2. change this page url to the following `<a href="{% pageurl page %}#comments-list">` which links users directly to the element with the id `comments-list`